contracts/SharerV3.sol

```diff
@@ -70,14 +70,14 @@ contract SharerV3 {
     function distribute(address _strategy) public{
         IStrategy strategy = IStrategy(_strategy);
         IERC20 reward =  IERC20(strategy.vault());
-        Contributor[] memory contributorsT = shares[_strategy];
+        if(reward.balanceOf(_strategy) <= 1000){
```

Comment:
doesnt moving this above totalRewards declaration cost more gas as we now have to read reward.balanceOf(_strategy) twice?